wear/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java

```diff
@@ -52,7 +52,7 @@
         INITIAL
     }
 
-    private static final boolean testWithDeadSensor = true; // never in production
+    private static final boolean testWithDeadSensor = false; // never in production
```

Comment:
Is this change intentional?

Reply:
This is a bugfix, actually already fixed in https://github.com/NightscoutFoundation/xDrip/pull/447